Tests/Support/Snapshots/edgecases-int32-int64/Sources/Entities/ArrayTest.swift

```diff
@@ -6,10 +6,10 @@ import NaiveDate
 
 public struct ArrayTest: Codable {
     public var arrayOfString: [String]?
-    public var arrayArrayOfInteger: [[Int64]]?
+    public var arrayArrayOfInteger: [[Int]]?
```

Comment:
Can you please update `testEdgecasesEnableIntegerCapacity()` so use `dataTypes` and re-record, that'll undo this diff 🙏